DEV-Simple/Utilities/MediaManager.swift

```diff
@@ -1,38 +1,61 @@
-//
-//  PodcastDelegate.swift
-//  DEV-Simple
-//
-//  Created by Fernando Valverde on 2/26/20.
-//  Copyright © 2020 DEV. All rights reserved.
-//
-
 import UIKit
-import WebKit
-import Foundation
-import AVFoundation
+import AVKit
 import MediaPlayer
 
 class MediaManager: NSObject {
 
-    weak var webView: WKWebView?
+    weak var webView: DEVWebView?
     var devToURL: String
 
     var avPlayer: AVPlayer?
+    var playerItem: AVPlayerItem?
+    var currentStreamURL: String?
 
-    var currentPodcast: AVPlayerItem?
-    var currentPodcastURL: String?
     var episodeName: String?
     var podcastName: String?
     var podcastRate: Float?
     var podcastVolume: Float?
     var podcastImageUrl: String?
     var podcastImageFetched: Bool = false
 
-    init(webView: WKWebView, devToURL: String) {
+    init(webView: DEVWebView, devToURL: String) {
         self.webView = webView
         self.devToURL = devToURL
     }
 
+    func loadVideoPlayer(videoUrl: String?, seconds: String?) {
+        if currentStreamURL != videoUrl, let videoUrl = videoUrl, let url = NSURL(string: videoUrl) {
+            currentStreamURL = videoUrl
+            playerItem = AVPlayerItem.init(url: url as URL)
+            avPlayer = AVPlayer.init(playerItem: playerItem)
+            avPlayer?.volume = 1.0
```

Comment:
Should we be cranking up the volume like this?

Reply:
Makes sense to be cautious about this! But it doesn't have a direct impact to the user. 

This is the volume of the `AVPlayer`, which is similar to the volume of the YouTube video player on the web. Your system volume is what will dictate how loud the media will play, but internal controls can increase or lower the player's volume. For example our own Podcast player has a volume slider (I think this can bee seen on the web/ipads but not smaller screens like iphones).

In the end this is okay and won't result in blasting the videos at full volume to users 😃